testsuite/features/secondary/proxy_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,175 @@
+    And I press "Add Item" in configured zones section
+    And I enter "tf.local" in third configured zone name field
+    # direct zone tf.local:
+    #   (Avahi does not cross networks, so we need to cheat by serving tf.local)
+    And I scroll to the top of the page
+    And I press "Add Item" in available zones section
+    And I enter "tf.local" in third available zone name field
+    And I enter "master/db.tf.local" in third file name field
+    And I enter the hostname of "proxy" in third name server field
+    And I enter "admin@tf.local." in third contact field
+    And I press "Add Item" in third A section
+    And I enter the hostname of "proxy" in fifth A name field
+    And I enter the IP address of "proxy" in fifth A address field
+    And I press "Add Item" in third A section
+    And I enter the hostname of "server" in sixth A name field
+    And I enter the IP address of "server" in sixth A address field
+    And I press "Add Item" in third NS section
+    And I enter the hostname of "proxy" in third NS field
+    # end
+    And I scroll to the top of the page
+    And I should see a "Bind" text
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Apply the highstate after the formula setup
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "States" in the content area
+    And I click on "Apply Highstate"
+    And I wait until event "Apply highstate scheduled by admin" is completed
+
+  Scenario: Create auto installation distribution
+    And I install package tftpboot-installation on the server
+    And I wait for "tftpboot-installation-SLE-15-SP2-x86_64" to be installed on "server"
+    And Ensure the "server" resolves its own public address
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "Create Distribution"
+    And I enter "SLE-15-SP2-TFTP" as "label"
+    And I enter "/usr/share/tftpboot-installation/SLE-15-SP2-x86_64/" as "basepath"
+    And I select "SLE-Product-SLES15-SP2-Pool for x86_64" from "channelid"
+    And I select "SUSE Linux Enterprise 15" from "installtype"
+    And I enter "useonlinerepo insecure=1" as "kernelopts"
+    And I click on "Create Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distributions" text
+    And I should see a "SLE-15-SP2-TFTP" link
+
+  Scenario: Create auto installation profile
+    And I follow the left menu "Systems > Autoinstallation > Profiles"
+    And I follow "Upload Kickstart/Autoyast File"
+    When I enter "15-sp2-cobbler" as "kickstartLabel"
+    And I select "SLE-15-SP2-TFTP" from "kstreeId"
+    And I attach the file "/sle-15-sp2-autoyast.xml" to "fileUpload"
+    And I click on "Create"
+    Then I should see a "Autoinstallation: 15-sp2-cobbler" text
+    And I should see a "Autoinstallation Details" text
+    When I enter "self_update=0" as "kernel_options"
```

Comment:
I would split it at that point and create another scenario, keep into account that we keep the session and the current page you let in the previous scenario.

Reply:
This was stolen from the minkvm test: I prefer to refactor both tests together in a separated card.

Reply:
Ok. Could you create another card for that one too please?

Reply:
Sure, I am going to create both.